core/controllers/reader.py

```diff
@@ -300,7 +300,123 @@ def get(self, exploration_id):
 
 
 class StorePlaythroughHandler(base.BaseHandler):
-    """Handles a useful playthrough coming in from the frontend to store it."""
+    """Handles a useful playthrough coming in from the frontend to store it. If
+    the playthrough already exists, it is updated in the datastore.
+    """
+
+    def __init__(self, *args, **kwargs):
+        """Method that initializes member variables for the handler.
+
+        Attributes:
+            exp_issues: ExplorationIssues. The exploration issues domain object.
+            issue_schema_version: int. The issue schema version.
+        """
+        super(StorePlaythroughHandler, self).__init__(*args, **kwargs)
+        self.exp_issues = None
+        self.issue_schema_version = None
+        self.playthrough_id = None
```

Comment:
Please prefix these with `current_`. The current names suggest that they will never change, but that's not true.

Reply:
Done.